Packs/PrismaAccess/Integrations/PrismaSASE/PrismaSASE.yml

```
@@ -0,0 +1,1891 @@
+    - default: false
+      isArray: false
+      name: page_size
+      description: Number of results per page to display.
+    - description: Tenant services group ID. If not provided, the tsg_id integration parameter will be used as the default.
+      name: tsg_id
+    deprecated: false
+    description: Lists all addresses objects.
+    execution: false
+    name: prisma-sase-address-object-list
+    outputs:
+    - contextPath: PrismaSase.Address.description
+      description: Address description.
+      type: String
+    - contextPath: PrismaSase.Address.folder
+      description: Address folder.
+      type: String
+    - contextPath: PrismaSase.Address.id
+      description: Address ID.
+      type: String
+    - contextPath: PrismaSase.Address.name
+      description: Address name.
+      type: String
+    - contextPath: PrismaSase.Address.address_value
+      description: Address value.
+      type: String
+    - contextPath: PrismaSase.Address.type
+      description: Address type.
+      type: String
+  - arguments:
+    - default: false
+      description: A specific tag to return. If not specified, all tags will be returned.
+      isArray: false
+      name: tag_id
+      required: false
+    - auto: PREDEFINED
+      default: false
+      defaultValue: Shared
+      description: Prisma sase folder location.
+      isArray: false
+      name: folder
+      predefined:
+      - Shared
+      - Mobile Users
+      - Remote Networks
+      - Service Connections
+      - Mobile Users Container
+      - Mobile Users Explicit Proxy
+      required: false
+    - defaultValue: '50'
+      description: The maximum number of results to return. Default is 50.
+      isArray: false
+      name: limit
+      required: false
+    - default: false
+      isArray: false
+      name: page
+      description: Page number you would like to view. Each page contains page_size values. Must be used along with page_size.
+    - default: false
+      isArray: false
+      name: page_size
+      description: Number of results per page to display.
+    - description: Tenant services group ID. If not provided, the tsg_id integration parameter will be used as the default.
+      name: tsg_id
+    deprecated: false
+    description: Lists all tags.
+    execution: false
+    name: prisma-sase-tag-list
+    outputs:
+    - contextPath: PrismaSase.Tag.id
+      description: Tag ID.
+      type: String
+    - contextPath: PrismaSase.Tag.name
+      description: Tag name.
+      type: String
+    - contextPath: PrismaSase.Tag.folder
+      description: Tag folder.
+      type: String
+    - contextPath: PrismaSase.Tag.comments
+      description: Tag comments.
+      type: String
+    - contextPath: PrismaSase.Tag.color
+      description: The tag color.
+      type: String
+  - arguments:
+    - auto: PREDEFINED
+      default: false
+      defaultValue: Shared
+      description: Prisma sase folder location.
```

```suggestion
      description: Prisma SASE folder location.
```
Is this the folder location or the folder type/name?